modules/codemod/lib/v7/utils/getImportRenameMap.ts

```diff
@@ -12,7 +12,9 @@ export function getImportRenameMap(j: JSCodeshift, root: Collection<any>, packag
     const value = node.source.value;
     if (
       typeof value === 'string' &&
-      (value.startsWith('@workday/canvas-kit-react') || value === packageName)
+      (value === '@workday/canvas-kit-react' ||
+        value.startsWith('@workday/canvas-kit-react/') ||
+        value === packageName)
```

Comment:
This change prevents old imports from matching. For example `@workday/canvas-kit-react-core`